src/foundry/foundry.js/clientDocumentMixin.d.ts

```diff
@@ -333,103 +334,54 @@ export declare class ClientDocumentMixin<T extends foundry.abstract.Document<any
    * Remove any features of the data which are world-specific.
    * This function is asynchronous in case any complex operations are required prior to exporting.
    * @param pack - A specific pack being exported to
+   *               (unused)
+   * @param options - Additional options which modify how the document is converted
+   *                  (default: `{}`)
    * @returns A data object of cleaned data suitable for compendium import
    */
-  toCompendium(pack?: CompendiumCollection<CompendiumCollection.Metadata>): Omit<
-    T['data']['_source'],
-    '_id' | 'folder' | 'permission'
-  > & {
+  toCompendium(
+    pack?: CompendiumCollection<CompendiumCollection.Metadata>,
```

Comment:
Apparently, it's also possible to pass `null` for `pack`. it happens in `exportToJSON`